pkg/cmd/server/server.go

```diff
@@ -212,23 +218,25 @@ func NewCommand(f client.Factory) *cobra.Command {
 }
 
 type server struct {
-	namespace             string
-	metricsAddress        string
-	kubeClientConfig      *rest.Config
-	kubeClient            kubernetes.Interface
-	veleroClient          clientset.Interface
-	discoveryClient       discovery.DiscoveryInterface
-	discoveryHelper       velerodiscovery.Helper
-	dynamicClient         dynamic.Interface
-	sharedInformerFactory informers.SharedInformerFactory
-	ctx                   context.Context
-	cancelFunc            context.CancelFunc
-	logger                logrus.FieldLogger
-	logLevel              logrus.Level
-	pluginRegistry        clientmgmt.Registry
-	resticManager         restic.RepositoryManager
-	metrics               *metrics.ServerMetrics
-	config                serverConfig
+	namespace                        string
+	metricsAddress                   string
+	kubeClientConfig                 *rest.Config
+	kubeClient                       kubernetes.Interface
+	veleroClient                     clientset.Interface
+	csiSnapClient                    snapshotvebeta1client.Interface
+	discoveryClient                  discovery.DiscoveryInterface
+	discoveryHelper                  velerodiscovery.Helper
+	dynamicClient                    dynamic.Interface
+	sharedInformerFactory            informers.SharedInformerFactory
+	snapshotterSharedInformerFactory *CSIInformerFactoryWrapper
```

Comment:
suggest using `CSI` somewhere in the name of this field.
may be `csiSnapshotterShareInformerFactory`